src/shogun2-core/src/test/resources/META-INF/spring/mail-test-context.xml

```diff
@@ -11,7 +11,7 @@
 
     <context:property-placeholder location="classpath*:META-INF/*.properties" />
 
-    <context:component-scan base-package="de.terrestris.shogun2.util" />
+    <context:component-scan base-package="de.terrestris.shogun2.util.mail" />
```

Comment:
OK for me. I was looking through the diff, not reading the commit message. Thanks for the clarification.